### DISCUSSION TITLE
Bug: wrong week number when using weekStartsOn

### DISCUSSION THREAD

**Discussion by pahen** [gpbl/react-day-picker#1811] [Support] [answered]
### Bug description

Seems like the week number is computed wrong when using weekStartsOn={1}. At least according to the ISO-8601 standard. https://www.epochconverter.com/weeknumbers says it 24 today but the react-day-picker says it's 25.

### To reproduce

https://codesandbox.io/s/day-picker-sandbox-forked-jdx2my?file=/src/Calendar.js

**Comment by gpbl** [comment, marked as answer]
Set the `ISOWeek` prop to switch to the ISO Week Date: https://codesandbox.io/s/day-picker-sandbox-forked-4zpvn6?file=/src/Calendar.js

I understand this was an option hidden in the docs, so I've added more details:
https://react-day-picker.js.org/basics/customization#switching-to-iso-week-dates. Thanks for the report!

**Reply by pahen** [reply]
Thanks! Works perfectly 🥇
